src/main/java/calculator/StringTokenizer.java

```diff
@@ -0,0 +1,19 @@
+package calculator;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringTokenizer {
+    public static final String DEFAULT_DELIMITERS = "[,:]";
+    public static final String CUSTOM_DELIMITER_REGEX = "//(.)\n(.*)";
```

Comment:
상수 활용 👍 
Pattern 객체를 상수로 선언해 보셔도 좋을 것 같아요
```suggestion
    public static final Pattern CUSTOM_DELIMITER_PATTERN = Pattern.compile(CUSTOM_DELIMITER_REGEX);
```

Reply:
객체를 상수로 선언할 수도 있군요! 🙌